subsys/bluetooth/audio/Kconfig.tbs

```
@@ -152,4 +152,63 @@ config BT_DEBUG_TBS
 
 endif # BT_TBS
 
+
+##################### Call Control Client #####################
+
+config BT_TBS_CLIENT
+	bool "Telephone Bearer Service client"
+	select BT_GATT_CLIENT
+	select BT_GATT_AUTO_DISCOVER_CCC
+	help
+	  This option enables support for Telephone Bearer Service client.
+
+if BT_TBS_CLIENT
+
+config BT_TBS_CLIENT_GTBS
+	bool "Telephone Bearer Service client GTBS support"
+	default y
+	help
+	  This option enables support for the generic TBS for the
+	  Call Control client.
+
+config BT_TBS_CLIENT_MAX_CALLS
+	int "Telephone Bearer Service client Client Maximum Number Of Calls Supported"
```

We are already in `BT_TBS_CLIENT` and the following settings doesn't include `Telephone Bearer Service client` in their titles, so maybe just:
```suggestion
	int "Maximum Number Of Calls Supported"
```